docs/source/examples/workspaces/openstack-libvirt/README.rst

```diff
@@ -0,0 +1,28 @@
+Openstack
```

Comment:
Looks like the documentation needs to be updated

Reply:
@14rcole Forgot about that. 
I will update the PR with new documentation